databases-w1/homework1.sql

```diff
@@ -0,0 +1,53 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT(*)
+AS total_tasks
+FROM task;
+
+SELECT COUNT(*) FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+SELECT * FROM task WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * 
+FROM task
+WHERE status_id = 3;
```

Comment:
this only works if you know the ID beforehand, but its a valid solution, remember you can query by the exact data they provide you, in this case 'Done'

Reply:
WHERE status = 'Done';